setup.sh

```diff
@@ -48,18 +55,26 @@ setup_linux() {
         echo ""
         echo "We don't yet have automated setup for RPM-based distributions"
         echo "and would be absolutely delighted to take a patch."
+        exit 1
     fi
 
     echo "Installing dependencies"
     echo ""
     echo "I'm about to use sudo to install some libraries, python 3.9, and curl"
     echo "so will ask for your root password"
     echo ""
-    sudo apt-get install libbz2-dev liblzma-dev libreadline-dev libsqlite3-dev python3.9 curl
+    sudo apt-get install \
+        libbz2-dev \
+        liblzma-dev \
+        libreadline-dev \
+        libsqlite3-dev \
+        python3.9 \
+        python3-pip \
+        curl
 
-    curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python3.9 -
+    curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/install-poetry.py | python3.9 -
 
-    "$HOME"/.poetry/bin/poetry install --extras lint
+    "$HOME"/.local/bin/poetry install --extras lint
```

Comment:
Can you add the post install `echo` from the macos steps that suggest the first command to run?